### PR TITLE
Redefine `is_dependant`

### LINKED ISSUE #446: Redefine `is_dependant`
Redefine `is_dependant` to someone who is financially/behavior supported by someone else instead of age based

### PR DESCRIPTION
**Pull request by Shinuing**
Fixes #446

### CODE REVIEW

File: policyengine_canada/variables/household/person/is_dependant.py [hunk 1]
```diff
@@ -4,9 +4,10 @@
...
     value_type = bool
     entity = Person
-    label = "Is a dependant"
+    label = "Is a dependant in the household"
     definition_period = YEAR
 
-    # Impute dependant status on age.
     def formula(person, period, parameters):
-        return person("age", period) < 18
+        head = person("head", period)
```
Comment (PavelMakarchuk) [inline policyengine_canada/variables/household/person/is_dependant.py:11]:
```suggestion
        head = person("is_head", period)
```

File: policyengine_canada/variables/household/person/is_dependant.py [hunk 2]
```diff
@@ -4,9 +4,10 @@
...
+        spouse = person("spouse", period)
```
Comment (PavelMakarchuk) [inline policyengine_canada/variables/household/person/is_dependant.py:12]:
```suggestion
        spouse = person("is_spouse", period)
```

File: policyengine_canada/variables/household/person/is_dependant.py
Comment (PavelMakarchuk) [inline policyengine_canada/variables/household/person/is_dependant.py:1]: need a unit test

File: policyengine_canada/variables/household/person/is_dependant.py [hunk 3]
```diff
@@ -4,9 +4,10 @@
 class is_dependant(Variable):
     value_type = bool
     entity = Person
-    label = "Is a dependant"
+    label = "Is a dependant in the household"
```
Comment (PavelMakarchuk) [inline policyengine_canada/variables/household/person/is_dependant.py:7]:
```suggestion
    label = "Is a dependant"
```

**Review by MaxGhenis — APPROVED** [review]

**Pull request merged**